scripts/docstrings_checker.py

```diff
@@ -167,7 +172,13 @@ def docstringify(docstring):
     return _check_docs_utils.Docstring(docstring)
 
 
-class GoogleDocstring(_check_docs_utils.GoogleDocstring):
+# TODO(#15613): Here we use MyPy ignore because of the incomplete typing of
+# pylint library and absences of stubs in pylint, forces MyPy to
+# assume that BaseChecker class has attributes of type Any.
+# Thus to avoid MyPy's error
+# (Class cannot subclass 'BaseChecker' (has type 'Any')),
+# we added an ignore here.
+class GoogleDocstring(_check_docs_utils.GoogleDocstring):  # type: ignore[misc]
```

Comment:
Agree but please create a different issue and mention it here, because issue #15613 was filed for missing stubs of the apache beam library. ( or if you want to use the same issue, then please update the content of #15613 ).

Reply:
Just to close the conversation!

A new issue is created refer: https://github.com/oppia/oppia/pull/16548#discussion_r1025519799